### ISSUE TITLE
Documentation needed

### ISSUE THREAD

**Issue by housel** [clearpathrobotics/rosserial_leonardo_cmake#1]
This package does pretty much what I wanted it to, but without documentation it took a bit of effort to get get all the details right. Filling in the README would help considerably.

**Comment by mikepurvis** [comment]
I agree! It's been a low priority, but should be done in the next little while.

In the meantime, feel free to issue a PR or begin work on a ROS wiki page.
